tests/test_browser.py

```diff
@@ -2036,6 +2036,7 @@ def test_cubegeom_normal_dap_far_glda(self): # use glDrawArrays
     self.btest('cubegeom_normal_dap_far_glda.c', reference='cubegeom_normal_dap_far_glda.png', args=['-s', 'LEGACY_GL_EMULATION=1', '-lGL', '-lSDL'])
 
   @requires_graphics_hardware
+  @no_firefox('fails on CI but works locally')
```

Comment:
Might be nice to have a pattern of opening bugs and referring to them here when we disable tests like this.

Reply:
Yeah, I'm not sure where to open a bug though. We don't have a good enough STR for a firefox bug I think ("open a CircleCI instance.."), and likewise for CircleCI (who don't work on GL stuff).